server/public/model/utils.go

```diff
@@ -631,6 +631,12 @@ func IsValidEmail(email string) bool {
 		return false
 	}
 
+	// mail.ParseAddress accepts quoted strings for the address
+	// which can lead to sending to the wrong email address
```

Comment:
Can you `strings.ContainsAny(email, "\"") || strings.Count(email, "@") > 1` ? Also the comment will need updating

Reply:
Its unnecessary...but sure I can add it.